pkg/workflow/executor.go

```diff
@@ -0,0 +1,46 @@
+package workflow
+
+import (
+	"fmt"
+
+	"github.com/Azure/Orkestra/pkg/utils"
+	v1alpha12 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// The set of executor actions which can be performed on a helmrelease object
+	Install ExecutorAction = "install"
```

Comment:
Doesnt need to be exported if it isnt being used anywhere else

Reply:
lowercase version of this conflicts with other values so keeping as capitalized